lib/airbrake-ruby/async_sender.rb

```diff
@@ -74,11 +72,22 @@ def closed?
     # @return [Boolean] true if an instance wasn't closed, but has no workers
     # @see https://goo.gl/oydz8h Example of at_exit that prevents exit
     def has_workers?
+      if @pid != Process.pid && @workers.list.empty?
+        @pid = Process.pid
+        spawn_workers
+      end
```

Comment:
Also works. My thinking was that in this case `@pid` won't be updated, but since the sender is closed it doesn't matter. Updated according to your suggestion.